src/Windowing/OpenToolkit.GraphicsLibraryFramework/GLFW.cs

```diff
@@ -8,391 +8,4027 @@
 //
 
 using System;
-using AdvancedDLSupport;
-using OpenToolkit.Core.Loader;
+using System.Runtime.InteropServices;
+using static OpenToolkit.Core.MarshalUtility;
+using static OpenToolkit.GraphicsLibraryFramework.GLFWNative;
 
 namespace OpenToolkit.GraphicsLibraryFramework
 {
+    // TODO: Make a final pass over this API to ensure consistency in:
+    // * Usage of enums instead of integers.
+    // * In/out/ref parameters and equivalents taking pointers.
+    // * String methods having byte* alternatives.
+    // * Correct handling of nullability with string/ref marshalling!
+
```

Comment:
Dangling TODO here. Happy with these?

Reply:
Was planning to do it in another PR, since this one's already quite big.